packages/components/Notification/src/Notification.tsx

```diff
@@ -53,8 +53,8 @@ export const Notification = compose<NotificationType>({
     root: Pressable,
     icon: Icon,
     contentContainer: View,
-    title: Text,
-    message: Text,
+    title: Body2Strong,
+    message: Body2,
```

Comment:
I think I prefer the old styling better, AKA: Text as the slot and the variant set in slot props. Was there a reason for this change I'm perhaps missing?

Reply:
The reason is to get a full end-to-end test of the new V2 text components.

Reply:
Talked offline: Idea is to use the new component aliases E2E in a component. While it's not my personal preference, it does makes sense and there's nothing technically wrong with it so approving.